.github/workflows/set-milestone-on-referenced-issue.yml

```diff
@@ -0,0 +1,33 @@
+#############################################################################
+# GitHub action to set latest milestone on issue of merged PR.
+#
+#############################################################################
+name: 'Set milestone on issue of merged PR'
+
+on:
+  pull_request:
+    types:
+      - closed
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  set-milestone:
+    name: 'Set milestone on issue of merged PR #${{ github.event.pull_request.number }}'
+    if: ${{ github.event.pull_request.merged == true }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v3
```

Comment:
why we need to checkout code?
we do nothing on code.

Reply:
Yeah, but we are calling the script from the .ci folder. You are telling me that I do not need to checkout? Let me check...

Reply:
We need checkout becase we use script from repository. This is result without checkout:
https://github.com/stoyanK7/checkstyle/actions/runs/4305395386/jobs/7507771772#step:2:7
```
Run ./.ci/set-milestone-on-referenced-issue.sh 
/home/runner/work/_temp/eb88c660-faca-4f16-a292-6edaec872ad1.sh: line 1: ./.ci/set-milestone-on-referenced-issue.sh: No such file or directory
Error: Process completed with exit code 127.
```

Reply:
Very clear reason